examples/tfengine/modules/team.hcl

```diff
@@ -288,6 +288,12 @@ template "project_data" {
       }]
       healthcare_datasets = [{
         name = "healthcare-dataset"
+        consent_stores = [{
+          name = "consent-store"
+          labels = {
```

Comment:
Could you please add enable_consent_create_on_update and default_consent_ttl here so they show up in the example?